<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hamegpowersupplycontrol</class>
 <widget class="QMainWindow" name="hamegpowersupplycontrol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>170</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>214</red>
        <green>214</green>
        <blue>214</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Hameg power supply control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ButtonConnect">
    <property name="geometry">
     <rect>
      <x>565</x>
      <y>85</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect to Hameg</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Controler Status&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboSerialBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>45</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="control_label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Not connected</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>781</width>
      <height>266</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="5">
      <widget class="QLabel" name="volt_4">
       <property name="font">
        <font>
         <family>Noto Sans [unknown]</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0V</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="ButtonOn_channel_4">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>On</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Noto Sans [unknown]</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Channel 1</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="ButtonOn_channel_1">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>On</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="current_1">
       <property name="font">
        <font>
         <family>Noto Sans [unknown]</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0A</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="current_3">
       <property name="font">
        <font>
         <family>Noto Sans [unknown]</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0A</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="ButtonOn_channel_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>On</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="volt_3">
       <property name="font">
        <font>
         <family>Noto Sans [unknown]</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0V</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="volt_1">
       <property name="font">
        <font>
         <family>Noto Sans [unknown]</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0V</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_volt2">
       <property name="maximum">
        <double>32.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
       <property name="value">
        <double>7.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="ButtonOn_channel_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 0, 0);</string>
       </property>
       <property name="text">
        <string>On</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>Noto Sans [unknown]</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Channel 4</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_volt3">
       <property name="maximum">
        <double>32.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
       <property name="value">
        <double>7.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Noto Sans [unknown]</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Voltage limit</string>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_volt4">
       <property name="maximum">
        <double>32.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
       <property name="value">
        <double>7.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Noto Sans [unknown]</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Channel 3</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Noto Sans [unknown]</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Voltage limit</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="volt_2">
       <property name="font">
        <font>
         <family>Noto Sans [unknown]</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0V</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="current_2">
       <property name="font">
        <font>
         <family>Noto Sans [unknown]</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0A</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Noto Sans [unknown]</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Voltage limit</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Noto Sans [unknown]</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Channel 2</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_volt1">
       <property name="maximum">
        <double>32.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
       <property name="value">
        <double>7.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Noto Sans [unknown]</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Voltage limit</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QLabel" name="current_4">
       <property name="font">
        <font>
         <family>Noto Sans [unknown]</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0A</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="currentZero_1">
       <property name="text">
        <string>Set current to zero</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="currentZero_2">
       <property name="text">
        <string>Set current to zero</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QPushButton" name="currentZero_3">
       <property name="text">
        <string>Set current to zero</string>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <widget class="QPushButton" name="currentZero_4">
       <property name="text">
        <string>Set current to zero</string>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>ButtonOn_channel_1</zorder>
    <zorder>label</zorder>
    <zorder>ButtonOn_channel_2</zorder>
    <zorder>volt_1</zorder>
    <zorder>ButtonOn_channel_3</zorder>
    <zorder>ButtonOn_channel_4</zorder>
    <zorder>label_5</zorder>
    <zorder>label_8</zorder>
    <zorder>label_11</zorder>
    <zorder>volt_2</zorder>
    <zorder>volt_3</zorder>
    <zorder>volt_4</zorder>
    <zorder>current_1</zorder>
    <zorder>current_2</zorder>
    <zorder>current_3</zorder>
    <zorder>current_4</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>doubleSpinBox_volt1</zorder>
    <zorder>doubleSpinBox_volt2</zorder>
    <zorder>doubleSpinBox_volt3</zorder>
    <zorder>doubleSpinBox_volt4</zorder>
    <zorder>currentZero_1</zorder>
    <zorder>currentZero_2</zorder>
    <zorder>currentZero_3</zorder>
    <zorder>currentZero_4</zorder>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>781</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0">
     <item row="3" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Channel 3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="horizontalSlider_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="horizontalSlider_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Channel 2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_1">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>4.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Channel 4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Channel 1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="horizontalSlider_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="horizontalSlider_1">
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>376</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel 1 software limited to 4A for use with Rb oven</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
